src/util/path-helpers.ts

```diff
@@ -0,0 +1,11 @@
+import { relative as nativeRelative } from 'path';
+
+const isWin = process.platform === 'win32';
```

Comment:
What's the difference between `process.platform` and `os.platform()`?

Reply:
Nothing.